app/user/components/FormField.tsx

```diff
@@ -0,0 +1,30 @@
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+
+interface FormFieldProps {
+    id: string;
+    label: string;
+    type?: string;
+    register: any;
```

Comment:
maybe we can get rid of this any type
look at 
https://stackoverflow.com/questions/76803987/what-type-to-use-for-register-from-the-react-hook-form-register-typescript-type/77636341#77636341